docs/supported.md

```diff
@@ -559,6 +559,40 @@ Direct Input: $← ↑ → ↓ ↔ ↕ ↖ ↗ ↘ ↙ ↚ ↛ ↞ ↠ ↢ ↣ 
 
 Extensible arrows all can take an optional argument in the same manner<br>as `\xrightarrow[under]{over}`.
 
+## Derivatives
+
+The syntax closely emulates popular $LaTeX$ packages that provide derivatives,
+such as `physics`, `diffcoef`, and `derivative`.
```

Comment:
I'd rather see this follow one spec. Or perhaps it already does, and this sentence just needs rewording? Fine to link to other packages, but which one should be considered "correct" for this PR? We don't want to implement a brand new derivative behavior. Stated another way, if someone was to copy KaTeX code over to LaTeX, which `\usepackage` should they include? (This will also help review this PR; I'd rather not read the documentation of three different packages, though I've already spent some time skimming them all.)